### PR TITLE
Mobile screen

### PR DESCRIPTION
**Pull request by lahday**
Did some fixes on the mobile screen

### CODE REVIEW

File: src/components/CartFilled.vue [hunk 1]
```diff
@@ -8,13 +8,16 @@
...
-      <p> $125.00 x {{}}</p>
+      <p> $125.00 x 3 <span class="font-bold">$375.00</span></p>
     </div>
+     <div class="px-2 text-left text-gray-300" @click="deleteCart">
+      <font-awesome-icon icon="fa-solid fa-trash" />
+      </div>
    
   </div>
-  <button class="m-4 px-10 py-3 bg-orange rounded-md text-white"> Checkout </button>
+  <button class="my-4 mx-8 px-10 py-3 w-3/4 bg-orange rounded-md text-white font-bold"> Checkout </button>
```
Comment (victorraji) [inline src/components/CartFilled.vue:20]: [request] Can we have screenshots of the mobile view ?

File: src/components/CartFilled.vue [hunk 2]
```diff
@@ -23,6 +26,16 @@
...
 export default {
+  data () {
+    return {
+    showCart: false
+    }
+  },
+  methods : {
+    deleteCart () {
+
+    }
```
Comment (victorraji) [inline src/components/CartFilled.vue:37]: [question] Why is there an empty method here?

File: src/components/CartFilled.vue [hunk 3]
```diff
@@ -23,6 +26,16 @@
 
 
 export default {
+  data () {
+    return {
+    showCart: false
+    }
+  },
```
Comment (victorraji) [inline src/components/CartFilled.vue:33]: [question]If this is vue3 why are you using the Options API and not the composition API?

File: src/components/ProductImages.vue [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+<script>
+
+export default {
+
+}
+</script>
+
+
+<style scoped>
+.prd-img{
```
Comment (victorraji) [inline src/components/ProductImages.vue:19]: [Request] Pls use verbose class names for your CSS

**Review by victorraji — CHANGES_REQUESTED** [review]

File: src/components/ProductImages.vue (hunk 4 above)
Comment (lahday) [reply src/components/ProductImages.vue:19]: noted

File: src/components/CartFilled.vue (hunk 2 above)
Comment (lahday) [reply src/components/CartFilled.vue:37]: I have not finished working on the functionalities of the app, would be completing that.

File: src/components/CartFilled.vue (hunk 1 above)
Comment (lahday) [reply src/components/CartFilled.vue:20]: image
image

**Pull request merged**